content/en/docs/reference/access-authn-authz/kubelet-tls-bootstrapping.md

```diff
@@ -460,7 +460,7 @@ You have several options for generating these credentials:
 
 ## kubectl approval
 
-CSRs can be approved outside of the approval flows builtin to the controller
+CSRs can be approved outside of the approval flows built in to the controller
```

Comment:
built-in ?

Reply:
Seems `built into` is easier to understand here. Need confirm.